package.json

```diff
@@ -39,8 +39,10 @@
   },
   "devDependencies": {
     "@polkadot/dev": "^0.79.3",
+    "@types/jest": "^29.5.12",
     "@types/node": "^20.10.5",
     "i18next-scanner": "^4.4.0",
+    "jest": "^29.7.0",
```

Comment:
Why are we adding jest?

Reply:
compiler failing with `: Cannot find name 'it'. Do you need to install type definitions for a test runner? Try `npm i --save-dev @types/jest` or `npm i --save-dev @types/mocha`.` after i changed the node resolution

Reply:
I think this is causing problems in others files with the defined custom global jest types referenced in `@polkadot/dev-test`.